yal/onboarding.py

```diff
@@ -92,8 +94,15 @@ async def state_persona_emoji(self):
         )
 
     async def state_save_persona_emoji(self):
+        def extract_first_emoji(persona_emoji):
+            emojs_present = "".join(c for c in persona_emoji if c in EMOJI_DATA)
+            if emojs_present:
+                return emojs_present[0]
+            return ""
```

Comment:
Extracting only the first character has unintended results for emoji's with modifiers:

```
>>> extract_first_emoji("👍🏿")
'👍'
```

So we might need to take a different approach with this, as well as adding some unit tests for these use cases.

I think we can use the `emoji_list` function from the `emoji` library, which should do better:
```
>>> from emoji import emoji_list
>>> emoji_list("👍🏿foo😀😀")
[{'match_start': 0, 'match_end': 2, 'emoji': '👍🏿'}, {'match_start': 5, 'match_end': 6, 'emoji': '😀'}, {'match_start': 6, 'match_end': 7, 'emoji': '😀'}]
```

Reply:
@rudigiesler yeah I ran into this issue this morning :) and this was my fix! I didnt realise you had reviewed the code 😂